### PR TITLE
Update textfile.md

### PR DESCRIPTION
**Pull request by sssnnt**
Added some text
@hakanfremin

### CODE REVIEW

File: textfile.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 ## header
```
Comment (hakanfremin) [inline textfile.md:1]: fix this

File: textfile.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 
 ## another
```
Comment (hakanfremin) [inline textfile.md:4]: test test

**Review by hakanfremin — CHANGES_REQUESTED** [review]
fix stuff!

File: textfile.md (hunk 1 above)
Comment (sssnnt) [reply textfile.md:1]: Sure

**Review by hakanfremin — APPROVED** [review]
ok

**Pull request merged**